Update index.adoc (#4105)
vaadin/docs: articles/index.adoc

[.card.large]

> === Vaadin Community

Reviewer: If we hide the CTA link, we can formulate this heading to be more actionable.

```suggestion
=== Join the Forum
```

```suggestion
=== Join the Community
```

> Stuck on something? Need feedback on your implementation? The Vaadin Forum is the perfect place to ask questions, share ideas, and connect with other developers.

Reviewer: In general, I’d avoid asking questions, and rather use direct statements. Is it good to force the reader to pause and think for an answer to these questions before going to the pitch about our forum? I know it’s subtle, but these question marks always trigger me :) 

```suggestion
If you’re stuck or need feedback on your implementation, the Vaadin forum is the perfect place to ask questions, share ideas, and connect with other developers.
```

Author: I agree. The suggested change makes sense.

> link:/forum/[Join the forum]

Reviewer: Should it be **The Vaadin Forum** or **The Vaadin forum**? In the text, it’s the former, and in the link it’s the latter. Is that on purpose?

> link:/forum/[Join the forum]

Reviewer: Maybe we could hide the CTA links on this card and the "video course" card as well, like on all the other cards on the landing page.

```suggestion
[.sr-only]
link:/forum/[Join the forum]
```

Author: I suggest keeping the CTA links, as it's not clear those boxes are clickable.
